src/Sulu/Component/Rest/ListBuilder/ListBuilderInterface.php

```diff
@@ -206,7 +206,7 @@ public function setPermissionCheck(UserInterface $user, $permission);
      * Defines a constraint for the rows to return.
      *
      * @param FieldDescriptorInterface $fieldDescriptor The FieldDescriptor which is checked
-     * @param string $value The value the FieldDescriptor should have
+     * @param mixed $value The value the FieldDescriptor should have
```

Comment:
I think we should support only scalar int|float|string here nothing more. If IN is supported maybe array of them.